zer0-os/zOS
# refactor(notifications-mute): use channel label management to handle mute room tagging

### What does this do?
- replaces specific mute functions/calls with label management functions/calls to handle mute room tagging

### Why are we making this change?
- now that we have label management in place we can consolidate the handling of room tagging.

### How do I test this?
- run tests as usual.
- run the ui and test mute room tagging is working as expected.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
  
  

https://github.com/user-attachments/assets/bbb8b370-53fe-403a-828d-912373e8e280
